test/message.ts

```diff
@@ -0,0 +1,120 @@
+// Needed due to chai's should.exist
+/* tslint:disable: no-unused-expression */
+import * as Promise from 'bluebird';
+import * as chai from 'chai';
+import * as ChaiAsPromised from 'chai-as-promised';
+import 'mocha';
+import { Conversation, ConversationReference, Front, Message,
+	Status } from '../lib/index';
```

Comment:
I don't like doing that. I guess it's personal preference, but I far prefer saving the vertical real estate (and it's still easy to see what's being imported).